letsdrill.rb

```diff
@@ -1,13 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+case integer
+when 80..89
+  return "B"
+when 70..79
+  return "C"
+when 60..69
+  return "D"
+else
+  if integer > 90
+    return "A"
+  else
+  return "F"
+end
+end
```

Comment:
I considered that, but what range would be the max for an A? I was trying to consider edge cases where the student had extra credit and had a grade greater than 100.